test/middleware/authentication.middleware.unit.ts

```diff
@@ -29,4 +30,12 @@ describe("authentication middleware tests", () => {
     expect(mockAuthMiddleware).toHaveBeenCalledWith(expectedAuthMiddlewareConfig);
     expect(mockAuthReturnedFunction).toHaveBeenCalledWith(req, res, next);
   });
+
+  it("should not call authentication middleware when loading accessibility page", () => {
+    req.originalUrl = CONFIRMATION_STATEMENT + ACCESSIBILITY_STATEMENT;
+    authenticationMiddleware(req, res, next);
+
+    expect(mockAuthMiddleware).toBeCalled;
```

Comment:
i think here you'd want to expect authMiddleware not to be called. What might be happening is that the mockAuthMiddleware is still remembering calls from the previous test. What we should do is add a beforeEach(()=> {}). function after the describe.
In there we can say mockAuthMiddleware.mockClear()
also I would clear the req.originalUrl between tests, so something like req.originalUrl = ""

Reply:
I have included those function and can confirm the unit test is passing. Thanks